docs/django-admin/roles.md

```diff
@@ -9,6 +9,28 @@ our `user_group` model and run in a migration.
 
 For more details, refer to the [user group model](../../src/registrar/models/user_group.py).
 
+## Adding a user as analyst or granting full access via django-admin
+
+If a new team member has joined whose data is NOT included in fixtures,
```

Comment:
**Dumb question:** Would we want to add a link of where to go to make these changes (ie getgov-ENV.app.cloud.gov/admin)

(I'm assuming that this is doc is not just for dev use so just in case, but feel free to ignore)